testsuite/integration-arquillian/tests/base/src/test/java/org/keycloak/testsuite/adapter/servlet/DemoServletsAdapterTest.java

```diff
@@ -210,99 +212,104 @@
     @Rule
     public AssertEvents assertEvents = new AssertEvents(this);
 
+    private static final Consumer<AdapterConfig> GLOBAL_ADAPTER_CONFIG = config -> {
```

Comment:
Is there some instability in this test? I don't see it from the Jira? Should we link another Jira to the PR?

Reply:
I've found out the instabilities, which this PR should solve are related to whole platform and there's created another issue (see #8175 ). Thus, this is out of date and I'll remove it.